### PR TITLE
TEST:  use pytest tmpdir fixture

### PR DESCRIPTION
**Pull request by christianbrodbeck**
Avoid creating files in the repository during testing.

### CODE REVIEW

File: surfer/tests/test_utils.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import nose.tools as nt
 from numpy.testing import assert_array_almost_equal, assert_array_equal
 
+from surfer.testing import requires_fsaverage
```
Comment (agramfort) [inline surfer/tests/test_utils.py]: I would not make testing a public module.

**Comment by agramfort** [comment]
appveyor is not happy

**Comment by christianbrodbeck** [comment]
I think appveyor is generally unhappy, something about setting up the environment...

**Comment by larsoner** [comment]
Let me iterate in #244 to get AppVeyor working again then we can re-run this, since it touches some code that was specific to the windows platform.

**Comment by larsoner** [comment]
Sorry @christianbrodbeck I broke this a bit :(

Do you have time to rebase? If not I can do it and push

**Comment by christianbrodbeck** [comment]
Rebased and all green...

**Comment by agramfort** [comment]
thx @christianbrodbeck

**Pull request merged**